src/bp/ui-admin/src/translations/en.json

```diff
@@ -124,6 +124,17 @@
         "yesterday": "Yesterday"
       }
     },
+    "modules": {
+      "experimental": "Experimental Modules",
+      "experimentalWarning": "Modules below are experimental and their use in production is not recommended, since they may change drastically before their official release. Migration will not be automatic when upgrading to a newer Botpress version. Use at your own risk.",
+      "helpText": "Changing the status of a module will update the module status in botpress.config.json. If cluster mode is disabled and Botpress is not in production mode, the module will be mounted or unmounted",
```

Comment:
```suggestion
      "helpText": "Enabling/disabling a module will update its status in the configuration files. If running a cluster in production, a restart of each node will be required for the change to take effect. Otherwise, the change will be immediate.",
```